modules/odc-installing-web-terminal.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// web_console/odc-about-web-terminal.adoc
+
+[id="odc-installing-web-terminal_{context}"]
+= Installing the web terminal
+
+You can install the web terminal using the *Web Terminal* Operator listed in the OpenShift Container Platform OperatorHub. When you install the *Web Terminal*  Operator, the Custom Resources (CRs) required for the command line configuration, such as the `DevWorkspace` resource, are automatically installed along with the Operator.
+
+.Prerequisites
+* Access to an OpenShift Container Platform cluster using an account with `cluster-admin` permissions.
+
+.Procedure
+. In the *Administrator* perspective of the web console, navigate to *Operators → OperatorHub*.
+. Use the *Filter by keyword* box to search for `Web Terminal` Operator in the catalog. Click the *Web Terminal*  tile.
+. Read the brief description about the Operator on the *Web Terminal*  page, and click *Install*.
+. On the *Install Operator* page, retain the default values for all fields.
+. The *All namespaces on the cluster (default)* for the installation mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+. The *Automatic* approval strategy ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM).
+. The *Update Channel* enables installation of the latest release of the *Web Terminal* Operator.
+. Click *Install*, and then click the *View operator in the Installed Operators page* to see the Operator listed on the *Installed Operators* page.
+. After the Operator is installed, refresh your page to see the command line terminal icon on the upper right of the console.
```

Comment:
You have to create the DevWorkspace CR first then, user can see the Web Terminal Icon. We need to add that one more step here about creating DevWorkspace CR.

Reply:
DevWorkspace CR is created if it does not exist after users click on Web Terminal icon and choose target namespace.